etc/schema/types/block_result.sql

```diff
@@ -17,5 +17,6 @@ CREATE TYPE block_result AS (
     "size" int4,
     "gasLimit" int8,
     "gasUsed" int8,
+    "mixHash" bytea,         -- 32 bytes
```

Comment:
The column type should be `hash` instead of `bytea`, similarly to e.g. `block_result` columns.

Reply:
@rahul10-pu Please make sure to press the 'Resolve' button on discussions when they've been sorted.